src/ParseServer.js

```diff
@@ -177,16 +174,22 @@ class ParseServer {
       }
     }
 
-    if (verbose || process.env.VERBOSE || process.env.VERBOSE_PARSE_SERVER) {
-      configureLogger({level: 'silly', jsonLogs});
-    }
-
     const filesControllerAdapter = loadAdapter(filesAdapter, () => {
       return new GridStoreAdapter(databaseURI);
     });
     // Pass the push options too as it works with the default
     const pushControllerAdapter = loadAdapter(push && push.adapter, ParsePushAdapter, push || {});
+    
     const loggerControllerAdapter = loadAdapter(loggerAdapter, WinstonLoggerAdapter);
+
+    log.setLogger(loggerControllerAdapter.logger);
```

Comment:
Because we set logger here, could we move earlier and avoid the default logger in logging.js? Or are we always going to need a default logger?